moduleframework/environment_prepare/docker_prepare.py

```diff
@@ -32,6 +32,7 @@ class EnvDocker(CommonFunctions):
 
     def prepare_env(self):
         self.installTestDependencies()
```

Comment:
What about to document this one? I would prefer to show user, what are you installing? Does it make sense?